### PR TITLE
feat(EET): Implement easy event tracking

### PR DESCRIPTION
**Pull request by msohailhussain**
## Summary
This change updates track to always send an event without doing experiment/variation lookup.

## Test plan
Updated unit tests
Manually Verifying EET

### CODE REVIEW

**Comment by aliabbasrizvi** [comment]
Triggered tests here: https://travis-ci.com/optimizely/fullstack-sdk-compatibility-suite/builds/97751580

**Comment by aliabbasrizvi** [comment]
Tests passed.

File: OptimizelySDK/Event/Builder/Params.cs [hunk 1]
```diff
@@ -17,27 +17,28 @@ namespace OptimizelySDK.Event.Builder
...
+        public const string DECISIONS           = "decisions";
+        public const string ENRICH_DECISIONS    = "enrich_decisions";
+        public const string ENTITY_ID           = "entity_id";
+        public const string EVENT_FEATURES      = "eventFeatures";
+        public const string EVENT_ID            = "eventEntityId";
+        public const string EVENT_METRICS       = "eventMetrics";
+        public const string EVENT_NAME          = "eventName";
+        public const string EVENTS              = "events";
+        public const string EXPERIMENT_ID       = "experiment_id";
+        public const string IS_LAYER_HOLDBACK   = "isLayerHoldback";
```
Comment (aliabbasrizvi) [inline OptimizelySDK/Event/Builder/Params.cs]: This is unused and can be removed.

File: OptimizelySDK/Event/Builder/Params.cs [hunk 2]
```diff
@@ -17,27 +17,28 @@ namespace OptimizelySDK.Event.Builder
...
+        public const string PROJECT_ID          = "project_id";
+        public const string REVISION            = "revision";
+        public const string TIME                = "timestamp";
+        public const string TIMESTAMP           = "timestamp";
+        public const string USER_FEATURES       = "userFeatures";
```
Comment (aliabbasrizvi) [inline OptimizelySDK/Event/Builder/Params.cs]: This is unused and can be removed.

File: OptimizelySDK/Event/Builder/Params.cs [hunk 3]
```diff
@@ -17,27 +17,28 @@ namespace OptimizelySDK.Event.Builder
...
-        public const string ANONYMIZE_IP = "anonymize_ip";
+        public const string ACCOUNT_ID          = "account_id";
+        public const string ANONYMIZE_IP        = "anonymize_ip";
+        public const string CAMPAIGN_ID         = "campaign_id";
+        public const string CLIENT_ENGINE       = "client_name";
+        public const string CLIENT_VERSION      = "client_version";
+        public const string DECISIONS           = "decisions";
+        public const string ENRICH_DECISIONS    = "enrich_decisions";
+        public const string ENTITY_ID           = "entity_id";
+        public const string EVENT_FEATURES      = "eventFeatures";
```
Comment (aliabbasrizvi) [inline OptimizelySDK/Event/Builder/Params.cs]: This is unused and can be removed.

File: OptimizelySDK/Event/Builder/Params.cs [hunk 4]
```diff
@@ -17,27 +17,28 @@ namespace OptimizelySDK.Event.Builder
...
+        public const string EVENT_ID            = "eventEntityId";
```
Comment (aliabbasrizvi) [inline OptimizelySDK/Event/Builder/Params.cs]: This is unused and can be removed.

File: OptimizelySDK/Event/Builder/Params.cs [hunk 5]
```diff
@@ -17,27 +17,28 @@ namespace OptimizelySDK.Event.Builder
...
+        public const string EVENT_METRICS       = "eventMetrics";
```
Comment (aliabbasrizvi) [inline OptimizelySDK/Event/Builder/Params.cs]: This is unused and can be removed.

File: OptimizelySDK/Event/Builder/Params.cs [hunk 6]
```diff
@@ -17,27 +17,28 @@ namespace OptimizelySDK.Event.Builder
...
+        public const string EVENT_NAME          = "eventName";
```
Comment (aliabbasrizvi) [inline OptimizelySDK/Event/Builder/Params.cs]: This is unused and can be removed. May be change this to be "key" and use it in code?

Comment (msohailhussain) [reply OptimizelySDK/Event/Builder/Params.cs]: removed it.

**Comment by aliabbasrizvi** [comment]
Tests passed. Merging

**Pull request merged**